src/main/app/src/app/shared/material-form-builder/form-components/documenten-lijst/documenten-lijst.component.ts

```diff
@@ -21,11 +21,12 @@ import {TranslateService} from '@ngx-translate/core';
 export class DocumentenLijstComponent extends FormComponent implements OnInit {
 
     data: DocumentenLijstFormField;
-    columns: string[] = ['select', 'titel', 'documentType', 'status', 'versie', 'auteur', 'creatiedatum', 'bestandsomvang', 'url'];
+    columns: string[] = ['select', 'titel', 'documentType', 'status', 'versie', 'auteur', 'creatiedatum', 'bestandsomvang', 'ondertekenen', 'url'];
     selection = new SelectionModel<EnkelvoudigInformatieobject>(true, []);
```

Comment:
Ondertekenen is een werkwoord. Dat is meer geschikt voor een method. Bedoel je misschien ondertekeningen in de geval?